### ISSUE TITLE
fbx loader load error

### ISSUE THREAD

**Issue by shadowredwine** [mrdoob/three.js#22167]
**Describe the bug**

An error occurs when reading some fbx files through the fbx loader.

**To Reproduce**

Reproducing the error is very simple.
Just open the attached fbx file with three.js editor.
The error is that the mesh and bone do not match.
Blender reads without problems.

**Platform:**

 - Device: Desktop,
 - OS: Desktop,
 - Browser: Chrome
 - Three.js version: 13
[Bone_Warrior.zip](https://github.com/mrdoob/three.js/files/6859573/Bone_Warrior.zip)
![화면 캡처 2021-07-22 110214](https://user-images.githubusercontent.com/87662232/126581021-4a530457-2e43-4db9-9684-960ccb5c6b68.png)

**Comment by Mugen87** [comment]
Where does the FBX file originally come from?

Since you can import the model to Blender, do you mind exporting to glTF and use `THREE.GLTFLoader`?

**Comment by shadowredwine** [comment]
model from asset store.
Reading the fbx file with blender and GLTF results in poor quality.
![화면 캡처 2021-07-22 172113](https://user-images.githubusercontent.com/87662232/126609597-f7c733f2-7034-4a38-aa26-885fecc05dbc.png)

**Comment by Mugen87** [comment]
Can you also try to convert the FBX asset to glTF via https://github.com/facebookincubator/FBX2glTF?

**Comment by Mugen87** [comment]
> Reading the fbx file with blender and GLTF results in poor quality.

What does "poor quality" mean? Can you share the resulting gltf asset in this thread?

**Comment by shadowredwine** [comment]
sorry. My english skill is not good enough.
If you read the fbx file through Blender and convert it to a gltf file, it feels like the 3d modeling quality is lowered.
Looking again, there seems to be a problem with the texture mapping.
The reason I thought so was because the texture files were not bundled together, so the mapping information seems to have been omitted.
[Bone Warrior.zip](https://github.com/mrdoob/three.js/files/6861041/Bone.Warrior.zip)

**Comment by Mugen87** [comment]
Um, the glTF asset actually looks good. If you can manage to fix the texture coordinates issue in Blender, I would recommend you choose the glTF workflow. For web based asset delivery it's a better choice than FBX anyway.

**Comment by shadowredwine** [comment]
It doesn't seem to be a mapping issue. When I checked, there was mapping information in Blender.
If you put the FBX file and the texture file in the same folder, blender automatically reads them. It seems to be an export problem with blender itself. In my opinion, FBX files should be read in FBX so there is no problem.

**Comment by Mugen87** [comment]
Yeah, it would be good to fix the issue in `FBXLoader`. But nevertheless, you should be aware that the more modern glTF and not FBX is the recommended 3D format of `three.js`. If you have to possibility, use it whenever you can. It's more compact, faster to parse and way easier to optimize (resulting in even smaller file sizes).

**Comment by shadowredwine** [comment]
It looked good to me too. More than anything, it was json, so it was easy to understand the structure. Please refer to the related links.
